src/otx/core/model/instance_segmentation.py

```diff
@@ -403,6 +403,7 @@ def _customize_inputs(self, entity: InstanceSegBatchDataEntity) -> dict[str, Any
         ):
             # NOTE: ground-truth masks are resized in training, but not in inference
             height, width = img_info.img_shape if self.training else img_info.ori_shape
+            mmdet_masks: BitmapMasks | PolygonMasks
```

Comment:
Why do we need this separately?

Reply:
I need to add type hints for mypy to recognize.